ci/install-build-deps.sh

```diff
@@ -16,7 +16,7 @@ case "$OS:$ARCH" in
 
         yum install -y centos-release-scl epel-release
         yum install -y \
-            autoconf autoconf-archive automake clang curl devtoolset-9-gcc devtoolset-9-gcc-c++ \
```

Comment:
I don't recall removing this, but perhaps it isn't needed anyway if the build was alread successful in `main`.

Reply:
Yeah, and the gates pass here as well...